<launch>
    <param name="robot_description" textfile="$(find cogimon_controller)/urdf/g1_29dof.urdf"/>
    <param name="robot_description_semantic" textfile="$(find cogimon_controller)/srdf/g1_29dof.srdf"/>

    <node pkg="cogimon_controller" type="joint_state_publisher" name="joint_state_publisher" output="screen">
        <param name="rate" value="1000"/>
    </node>
</launch>
